client/index.html

```diff
@@ -19,62 +19,20 @@
 <body>
     <div id="setup">
         <button id="botVsBotWhite">Bot Plays Bot</button>
-        <button id="playerVsBotWhite">Play Against Bot</button>
-        <button id="createBot">Make New Bot</button>
+        <button id="playerVsBotWhite" onclick='window.location.href="/choose_bot.html"' >Play Against Bot</button>
+        <button id="createBot" onclick='window.location.href="/editBot"'>Make New Bot</button>
+        <button id="editBot" onclick='window.location.href="/choose_bot_edit.html"'>Edit a Bot</button>
     </div>
 
     <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
```

Comment:
oh wait now that I'm ready what jquery is. It does seem like it could make our coding more convenient in these html files. I have not been using it since I hadn't look into it though